includes/admin/settings/class-wpsms-settings.php

```diff
@@ -2642,8 +2632,14 @@ public function render_settings($default = "general", $args = array())
 
                         ?>
 
-                        <?php if (isset($_GET['page']) and in_array($_GET['page'], ['wp-sms-integrations'])) {
-                            echo \WP_SMS\Helper::loadTemplate('zapier-section.php'); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+                        <?php if (isset($_GET['page']) && $_GET['page'] === 'wp-sms-integrations') {
+                            /**
+                             * Hook: wp_sms_integrations_box
+                             *
+                             * Fires inside the integrations page.
+                             * Allows WP SMS Pro (or other extensions) to output HTML.
+                             */
+                            echo wp_kses_post(apply_filters('wp_sms_integrations_box', ''));
```

Comment:
The filter name is not good, something like `wp_sms_settings_integrations_box` is much better.